libs/model-ad/api-description/src/components/schemas/DiseaseCorrelationSearchQuery.yaml

```diff
@@ -25,12 +25,18 @@ properties:
     maxItems: 2
     example: ['CONSENSUS NETWORK MODULES', 'Consensus Cluster A - ECM Organization']
   items:
-    description: List of item IDs to filter by.
+    description: |
+      List of composite identifiers to filter by. Each identifier uses the format "name~age~sex"
+      where each identifier represents one complete combination of model name, age, and sex.
+
+      Example: "APOE4~4 months~Female" filters for documents matching that exact name, age, and sex.
+      Multiple items can be provided to filter for multiple specific combinations.
```

Comment:
The search query should be an object to provide enough flexibility to extend it in the future with additional properties if needed.